packages/strapi-admin/files/public/app/app.js

```diff
@@ -115,34 +109,30 @@ import { pluginLoaded, updatePlugin } from './containers/App/actions';
  * @param params
  */
 const registerPlugin = (plugin) => {
+  const formattedPlugin = plugin;
```

Comment:
Why do you need to declare this variable? You could use the `plugin` variable reference instead